lib/ronad.rb

```diff
@@ -0,0 +1,4 @@
+require "ronad/version"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)